### ISSUE TITLE
Nested strings in interface

### ISSUE THREAD

**Issue by mjablecnik** [slang-i18n/slang#191]
I have this strings with interface:
```yaml
distribution_list(interface=DistributionType):
  point:
    title: "Seznam výdejních míst"
    error: "Nastala neočekávaná chyba: Výdejní místa nelze načíst."
    buttons:
      refresh: "Obnovit"
      finish: "OK"
  box:
    title: "Seznam boxů"
    error: "Nastala neočekávaná chyba: Seznam boxů nelze načíst."
    buttons:
      refresh: "Obnovit"
      finish: "Detail"
```

Into slang.yaml I added: 
![Screenshot 2024-02-05 at 14 55 21](https://github.com/slang-i18n/slang/assets/2686906/d2683c3e-16da-4aa3-94b5-b9869153b1f3)

During generation I have this:
![Screenshot 2024-02-05 at 14 55 49](https://github.com/slang-i18n/slang/assets/2686906/af4cbc42-cdeb-4307-a378-16ed6d3dfc75)

But generated code looks like this:
![Screenshot 2024-02-05 at 14 57 32](https://github.com/slang-i18n/slang/assets/2686906/5e6ed477-504f-4f58-a15a-d10b96d87664)

![Screenshot 2024-02-05 at 14 58 28](https://github.com/slang-i18n/slang/assets/2686906/8238c98f-1d11-4526-9ec1-3b3110198c93)

Can you somehow fix the generated code? Now it doesn't work.

**Comment by Tienisto** [comment]
Thanks for this issue. This is seems to be not supported yet.

Just a remark: You don't need to specify the interfaces in the config anymore, adding the hints in the i18n file is enough.

**Comment by Tienisto** [comment]
Fixed in https://github.com/slang-i18n/slang/commit/ba487b798427d8982cb630dca46b3cf572b0499e

You will need to specify the nested interface either via config or in every child:

```yaml
distribution_list(interface=DistributionType):
  point:
    title: "Seznam výdejních míst"
    error: "Nastala neočekávaná chyba: Výdejní místa nelze načíst."
    buttons(singleInterface=DistributionButton):
      refresh: "Obnovit"
      finish: "OK"
  box:
    title: "Seznam boxů"
    error: "Nastala neočekávaná chyba: Seznam boxů nelze načíst."
    buttons(singleInterface=DistributionButton):
      refresh: "Obnovit"
      finish: "Detail"
```

or as config:

```yaml
interfaces:
  DistributionButton:
    attributes:
      - String refresh
      - String finish
```

**Comment by Tienisto** [comment]
Published in v3.30.0
